lib/active_stash/reflection.rb

```diff
@@ -0,0 +1,20 @@
+module ActiveStash
+  class Reflection
```

Comment:
I don't understand the need for this class.  Why not just define these methods as public on the object you're passing into the constructor?  If that is a `CipherStash::Collection` (which it feels like it might be, given the instance variables you're poking at), I don't mind making these part of the public API, since they're obviously needed.

Reply:
This is a hack because these methods don't exist yet on CipherStash client. I'm very happy for them to go there instead if you have time!